mbed_greentea/mbed_target_info.py

```diff
@@ -297,3 +297,29 @@ def get_mbed_supported_test(mbed_test_case_name):
     @return Returns true if test case name from mbed SDK can be automated with mbed-greentea
     """
     return mbed_test_case_name not in NOT_SUPPORTED_TESTS
+
+
+def get_binary_type_for_platform(platform):
+    """
+    Gives binary type for the given platform.
+
+    :param platform:
+    :return:
+    """
+    return TARGET_INFO_MAPPING[platform]['properties']["binary_type"]
+
+
+def get_platform_property(platform, property):
+    """
+    Gives platform property.
+
+    :param platform:
+    :return:
+    """
+    if platform in TARGET_INFO_MAPPING:
+        if property in TARGET_INFO_MAPPING[platform]['properties']:
+            return TARGET_INFO_MAPPING[platform]['properties'][property]
+        else:
+            raise Exception("Property %s not found in platform %s" % (property, platform))
```

Comment:
The request for a ```property``` is made internally from greentea. It is not an external request. 
Question is what can be done if the property is not found. If nothing then ```Exception``` is right thing. At the moment this function has only one usage. That is for property ```"program_cycle_s"```. In the existing code I did not find any alternative to this hence an exception.